resources/install.rb

```diff
@@ -89,7 +90,7 @@
   as_sys_admin_list = build_admin_list(new_resource.as_sysadmins)
   dreplay_ctlr_admin_list = build_admin_list(new_resource.dreplay_ctlr_admins)
 
-  shared_wow_dir = new_resource.install_dir.gsub(/Program Files/, 'Program Files (x86)')
+#  shared_wow_dir = new_resource.install_dir.gsub(/Program Files/, 'Program Files (x86)')
```

Comment:
This should just get removed

Reply:
I can do it now, if you ask for a change?